language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/utils/NameUtil.java

```diff
@@ -214,6 +215,24 @@ public static String getModifiedTypeName(DocumentServiceContext context, TypeSym
         return CommonUtil.getModifiedSignature(context, typeSignature);
     }
 
+    /**
+     * Returns the record type.
+     *
+     * @param context Context
+     * @param wrapper A wrapper containing record type symbol and broader type symbol
+     * @return Record type
+     */
+    public static String getRecordType(DocumentServiceContext context,
```

Comment:
Maybe we can change this to `getRecordTypeName` WDYT? NameUtil javadoc - `Carries a set of utilities used for operations on names of variables, types, parameters, visible symbols.`

Reply:
getRecordTypeName is better. i'll change it